func3d.py

```diff
@@ -90,6 +90,17 @@ def pgradient(self, rvec):
         akderiv = 2*a/(1+a)**3 * r
         return akderiv
 
+def test(): 
+    import testwf 
+    pade = PadeFunction(0.2)
+    gauss = GaussianFunction(0.4)
+    for delta in [1e-3,1e-4,1e-5,1e-6,1e-7]:
+        print('Gaussian: delta', delta, "Testing gradient", testwf.test_func3d_gradient(gauss,delta=delta))
+        print('Gaussian: delta', delta, "Testing laplacian", testwf.test_func3d_laplacian(gauss,delta=delta))
+    for delta in [1e-3,1e-4,1e-5,1e-6,1e-7]:
```

Comment:
This is repeated code. You should loop as:
test_functions={'Gaussian':GaussianFunction(0.4)
'Pade':PadeFunction(0.2) 
}

Then loop over test_functions.items().